diff --git a/node_modules/better-sqlite3/bin/linux-x64-146/better-sqlite3.node b/node_modules/better-sqlite3/bin/linux-x64-146/better-sqlite3.node
new file mode 100755
index 0000000..b4c0daf
Binary files /dev/null and b/node_modules/better-sqlite3/bin/linux-x64-146/better-sqlite3.node differ
diff --git a/node_modules/better-sqlite3/build/Release/better_sqlite3.node b/node_modules/better-sqlite3/build/Release/better_sqlite3.node
deleted file mode 100755
index 8aa823d..0000000
Binary files a/node_modules/better-sqlite3/build/Release/better_sqlite3.node and /dev/null differ
diff --git a/node_modules/better-sqlite3/src/better_sqlite3.cpp b/node_modules/better-sqlite3/src/better_sqlite3.cpp
index 13f054a..d06f4eb 100644
--- a/node_modules/better-sqlite3/src/better_sqlite3.cpp
+++ b/node_modules/better-sqlite3/src/better_sqlite3.cpp
@@ -57,7 +57,11 @@ NODE_MODULE_INIT(/* exports, context */) {
 
 	// Initialize addon instance.
 	Addon* addon = new Addon(isolate);
+#if defined(V8_ENABLE_SANDBOX)
+	v8::Local<v8::External> data = v8::External::New(isolate, addon, v8::kExternalPointerTypeTagDefault);
+#else
 	v8::Local<v8::External> data = v8::External::New(isolate, addon);
+#endif
 	node::AddEnvironmentCleanupHook(isolate, Addon::Cleanup, addon);
 
 	// Create and export native-backed classes and functions.
diff --git a/node_modules/better-sqlite3/src/util/helpers.cpp b/node_modules/better-sqlite3/src/util/helpers.cpp
index f777152..db7fd9f 100644
--- a/node_modules/better-sqlite3/src/util/helpers.cpp
+++ b/node_modules/better-sqlite3/src/util/helpers.cpp
@@ -89,7 +89,7 @@ void SetPrototypeGetter(
 	recv->InstanceTemplate()->SetNativeDataProperty(
 		InternalizedFromLatin1(isolate, name),
 		func,
-		0,
+		nullptr,
 		data
 	);
 }
diff --git a/node_modules/better-sqlite3/src/util/macros.cpp b/node_modules/better-sqlite3/src/util/macros.cpp
index e75d537..d23654c 100644
--- a/node_modules/better-sqlite3/src/util/macros.cpp
+++ b/node_modules/better-sqlite3/src/util/macros.cpp
@@ -27,7 +27,11 @@
 #define EasyIsolate v8::Isolate* isolate = v8::Isolate::GetCurrent()
 #define OnlyIsolate info.GetIsolate()
 #define OnlyContext isolate->GetCurrentContext()
+#if defined(V8_ENABLE_SANDBOX)
+#define OnlyAddon static_cast<Addon*>(info.Data().As<v8::External>()->Value(v8::kExternalPointerTypeTagDefault))
+#else
 #define OnlyAddon static_cast<Addon*>(info.Data().As<v8::External>()->Value())
+#endif
 #define UseIsolate v8::Isolate* isolate = OnlyIsolate
 #define UseContext v8::Local<v8::Context> ctx = OnlyContext
 #define UseAddon Addon* addon = OnlyAddon
